### DISCUSSION TITLE
23 gennaio 2023 B - esercizio 1

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#26] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/b0df61cc-1a36-4b28-8220-f23b5ffdf032)

**Comment by FrascaHoryzont** [comment]
![Screenshot 2024-01-14 170829](https://github.com/sapienzastudentsnetwork/bdi2324/assets/100992294/d811159e-8654-45b4-aacc-a45283e8ac46)

**Reply by bobbylinux** [reply]
questa implementazione è errata, "Arrivo" non corrisponde al nome della città in Volo ma alla sigla aeroportuale (e una città può avere più aeroporti) deve essere per tanto messa in join VOLO con AEROPORTO e fatto una selezione del campo città di Aeroporto != o =  a Madrid.
Inoltre secondo me andrebbe fatta la differenza tra chi ha volato a Madrid e tutti gli altri, anche perché in personale ci possono essere anche dipendenti che non volano (ad esempio addetti al checkin), quindi la cosa migliore sarebbe "replicare" le query di NoMadrid e poi togliere quelli da chi ha volato a madrid per ottenere solo chi ha volato con destinazione Madrid

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/06d3d5dc-088c-4649-b8f1-ff2f738e38e2)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/1f945f94-56a9-4c40-b5b2-87f1d89a80e6)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/c02fbba6-bae6-4608-8179-269c45abe88b)

**Comment by bobbylinux** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/7393062/5d917efb-3ff4-4959-a4e9-515ae09bdd66)
